src/stylesheets/components/_footer.scss

```diff
@@ -1,130 +1,30 @@
 //
 // Footer Component
 //
-// Based on the existing GOV.UK footer in GOV.UK Template
+// GOV.UK Frontend footer adapted
 
 @include govuk-exports("app-footer") {
-  $app-footer-background: $govuk-grey-3;
-  $app-footer-border-top: #a1acb2;
-  $app-footer-link: #454a4c;
-  $app-footer-text: $app-footer-link;
-  $app-footer-link-hover: #171819;
-
-  .app-c-footer {
-    display: block;
-    padding: $govuk-gutter $govuk-spacing-scale-3 ($govuk-gutter * 2);
-    border-top: 1px solid $app-footer-border-top;
-    color: $app-footer-text;
-    background: $app-footer-background;
+  .govuk-c-footer--full .govuk-o-width-container {
+    max-width: 100%;
     @include mq($from: desktop) {
-      padding: $govuk-spacing-scale-6;
-    }
-
-    a:link,
-    a:visited {
-      color: $app-footer-link;
-    }
-
-    a:hover,
-    a:active {
-      color: $app-footer-link-hover;
+      margin: 0 auto;
     }
   }
 
-  .app-c-footer__navigation {
-
-    @include mq($until: desktop) {
-      margin-bottom: $govuk-spacing-scale-6;
+  .app-c-footer {
```

Comment:
I think this padding should be on the `app-o-width-container--full` class as described in the other comment.

Reply:
done